### PR TITLE
IDVA5-125-accessibility-page

### PR DESCRIPTION
**Pull request by abhiahirv1**
adding accessibility-page

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/types/pageURL.ts [hunk 1]
```diff
@@ -17,10 +19,13 @@ export const SOLE_TRADER_AUTO_LOOKUP_ADDRESS = SOLE_TRADER + "/correspondenceAdd
...
+
 export const SOLE_TRADER_ROLE = SOLE_TRADER + "/statement-relevant-officer";
 
 export const SOLE_TRADER_TYPE_OF_BUSINESS = SOLE_TRADER + "/type-of-business";
 
 export const SOLE_TRADER_OTHER_TYPE_OFBUSINESS = SOLE_TRADER + "/other-type-of-business";
 
 export const HEALTHCHECK = "/healthcheck";
+
+export const ACCESSIBILITY_STATEMENT = HELP + "/accessibility-statement";
```
Comment (swalke-ch) [inline src/main/types/pageURL.ts:31]: I think, we don't need /help in URL

File: src/main/views/features/help/accessibility-statement/accessibility-statement.njk [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+{% block main_content %}
+
+<main id="page-container" class="search govuk-width-container" role="main">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div id="search-container">
+        <article class="text" id="page-content">
+          <h1 class="govuk-heading-xl">
+            Accessibility statement for the Companies House service
```
Comment (swalke-ch) [inline src/main/views/features/help/accessibility-statement/accessibility-statement.njk:15]: I can just double check if there is any requirement of having this page in welsh ?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by atedros-ch — APPROVED** [review]
LGTM

**Pull request merged**